features/adb-openshift.feature

```diff
@@ -30,7 +38,12 @@ Feature: Command output from various OpenShift related commands in ADB
     ==> default: OpenShift service configured successfully...
     """
 
-    When I successfully run `bundle exec vagrant service-manager env openshift`
+    When I sleep for 10 seconds
+    And I successfully run `bundle exec vagrant service-manager status`
+    Then stdout from "bundle exec vagrant service-manager status" should contain "openshift - running"
+
+    When I sleep for 10 seconds
```

Comment:
This should not be the case. We need to investigate the root cause of why we need 10s delay.